picard/formats/apev2.py

```diff
@@ -80,42 +80,43 @@ class APEv2File(File):
-        "MixArtist": "remixer",
-        "Weblink": "website",
-        "DiscSubtitle": "discsubtitle",
-        "BPM": "bpm",
-        "ISRC": "isrc",
-        "CatalogNumber": "catalognumber",
-        "Barcode": "barcode",
-        "EncodedBy": "encodedby",
-        "Language": "language",
-        "MOVEMENT": "movementnumber",
-        "MOVEMENTNAME": "movement",
-        "MOVEMENTTOTAL": "movementtotal",
-        "SHOWMOVEMENT": "showmovement",
-        "MUSICBRAINZ_ALBUMSTATUS": "releasestatus",
-        "MUSICBRAINZ_ALBUMTYPE": "releasetype",
-        "musicbrainz_trackid": "musicbrainz_recordingid",
-        "musicbrainz_releasetrackid": "musicbrainz_trackid",
-        "Original Artist": "originalartist",
-        "REPLAYGAIN_ALBUM_GAIN": "replaygain_album_gain",
-        "REPLAYGAIN_ALBUM_PEAK": "replaygain_album_peak",
-        "REPLAYGAIN_ALBUM_RANGE": "replaygain_album_range",
-        "REPLAYGAIN_TRACK_GAIN": "replaygain_track_gain",
-        "REPLAYGAIN_TRACK_PEAK": "replaygain_track_peak",
-        "REPLAYGAIN_TRACK_RANGE": "replaygain_track_range",
-        "REPLAYGAIN_REFERENCE_LOUDNESS": "replaygain_reference_loudness",
+        "albumartist": "Album Artist",
+        "remixer": "MixArtist",
+        "website": "Weblink",
+        "discsubtitle": "DiscSubtitle",
+        "bpm": "BPM",
+        "isrc": "ISRC",
+        "catalognumber": "CatalogNumber",
+        "barcode": "Barcode",
+        "encodedby": "EncodedBy",
+        "language": "Language",
+        "movementnumber": "MOVEMENT",
+        "movement": "MOVEMENTNAME",
+        "movementtotal": "MOVEMENTTOTAL",
+        "showmovement": "SHOWMOVEMENT",
+        "releasestatus": "MUSICBRAINZ_ALBUMSTATUS",
+        "releasetype": "MUSICBRAINZ_ALBUMTYPE",
+        "musicbrainz_recordingid": "musicbrainz_trackid",
+        "musicbrainz_trackid": "musicbrainz_releasetrackid",
+        "originalartist": "Original Artist",
+        "replaygain_album_gain": "REPLAYGAIN_ALBUM_GAIN",
+        "replaygain_album_peak": "REPLAYGAIN_ALBUM_PEAK",
+        "replaygain_album_range": "REPLAYGAIN_ALBUM_RANGE",
+        "replaygain_track_gain": "REPLAYGAIN_TRACK_GAIN",
+        "replaygain_track_peak": "REPLAYGAIN_TRACK_PEAK",
+        "replaygain_track_range": "REPLAYGAIN_TRACK_RANGE",
+        "replaygain_reference_loudness": "REPLAYGAIN_REFERENCE_LOUDNESS",
     }
-    __rtranslate = dict([(v, k) for k, v in __translate.items()])
+    __rtranslate = dict([(v.lower(), k) for k, v in __translate.items()])
 
     def _load(self, filename):
         log.debug("Loading file %r", filename)
         file = self._File(encode_filename(filename))
         metadata = Metadata()
         if file.tags:
             for origname, values in file.tags.items():
-                if origname.lower().startswith("cover art") and values.kind == mutagen.apev2.BINARY:
+                origname = origname.lower()
+                if origname.startswith("cover art") and values.kind == mutagen.apev2.BINARY:
```

Comment:
I'd switch tests order here (but I'm not sure if Python optimizes this or not).

Reply:
ou are right, that's nicer. I switched that around, also same in the _save method.